tests/Unit/Core/Import/File/FileFinderTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace Tests\Unit\Core\Import\File;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Adapter\Configuration;
+use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopConstraint;
+use PrestaShop\PrestaShop\Core\Import\File\FileFinder;
+use PrestaShop\PrestaShop\Core\Import\ImportDirectory;
+use Symfony\Component\Filesystem\Filesystem;
+
+class FileFinderTest extends TestCase
+{
+    /** @var string */
+    protected $adminDirectory;
+
+    /** @var string */
+    protected $importSubDirectory;
+
+    /** @var FileFinder */
+    protected $filefinder;
+
+    /** @var ImportDirectory */
+    protected $importDirectory;
+
+    /** @var Filesystem  */
+    protected $fs;
+
+    protected function setUp(): void
+    {
+        $configuration = new ConfigurationMock();
```

Comment:
Usually we use `$this->createMock()` and redefine methods that needs to be mocked but I guess it works with an override 🤷‍♂️

Reply:
You're right, it's better. It's fixed :)